Dockerfile

```diff
@@ -1,14 +1,16 @@
 FROM node:18.18-alpine3.18
 
+RUN corepack enable
+
 WORKDIR /usr/src/app
 
 COPY . .
 RUN --mount=type=cache,target=/root/.yarn YARN_CACHE_FOLDER=/root/.yarn \
-    yarn install --network-timeout 600000 && \
+    yarn install --immutable --network-timeout 600000 && \
```

Comment:
Added the --immutable flag here as a precaution. The Docker build will fail if the Yarn install process would result in a different yarn.lock file, which could happen for example if two different versions of Yarn are used.